README.md

```
@@ -42,40 +1,240 @@
+실시간 예약 시스템과 직관적인 사용자 인터페이스를 통해 사용자와 관리자 모두 쉽게 사용할 수 있도록 개발하였습니다.
+
+<br />
+
+## 👥 Team
+
+<table align="center">
+    <tbody>
+        <tr>
+             <td>
+                <a href="https://github.com/miraclee1226">
+                    <img src="https://avatars.githubusercontent.com/miraclee1226" width="100" height="100"/>
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)